src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleProjectCleanupStep.java

```diff
@@ -57,4 +72,16 @@ protected StepResult cleanUp(CloudResourceUid resourceUid) {
       return new StepResult(StepStatus.STEP_RESULT_FAILURE_RETRY, e);
     }
   }
+
+  private static HttpRequestInitializer setHttpTimeout(
+      final HttpRequestInitializer requestInitializer) {
+    return new HttpRequestInitializer() {
+      @Override
+      public void initialize(HttpRequest httpRequest) throws IOException {
+        requestInitializer.initialize(httpRequest);
+        httpRequest.setConnectTimeout(3 * 60000); // 3 minutes connect timeout
+        httpRequest.setReadTimeout(3 * 60000); // 3 minutes read timeout
+      }
+    };
```

Comment:
I think using Google's HttpRequestInitializer is the right solution. I think this is in the realm of, "Google lets you configure this how you want, and it solves this problem. Use Google's solution: we don't need to wrap or build something for CRL."
For now, if this is the only place we're setting up a Cow that uses HttpRequestInitializer/the only place where we use CloudResourceManagerCow, it's ok. I could see us using dependency injection to coordinate HttpRequestInitializers in the long term, or other services.

Reply:
Yup, I think just do it here is okay for Janitor now. 
I am sure RBS will need DI for this.